WordPress/gutenberg
# Upgrade Lerna to the latest version

## What?

While doing the node/npm migration, I've noticed a few messages when installing packages. Among them, the fact that we're using a few deprecated packages. I'm trying to clean these out a little bit.

I noticed that our Lerna's dependency was responsible for 3 or 4 of these warnings, so in this PR I'm trying to upgrade it to the latest version.

## How?

Also, as recommended by Lerna's changelog. I've run `lerna repair` which updated the lerna.json file.

## Testing Instructions

It's a bit hard to test all the commands because some of them are only run when publishing packages. but I tried `npm run publish:check` and it seems to work as intended and we're also using some `lerna` in the build command which is run in CI.
